tests/test_inspect.py

```diff
@@ -80,6 +80,7 @@ def test_visualize_works(daa):
         dask.compute(query, optimize_graph=True)
 
 
+@pytest.mark.xfail()
```

Comment:
What happened here?

Reply:
Yikes! Good catch! That was in my local copy for some reason (not a recent reason, either).

```suggestion
```